docs/app/Examples/elements/Input/Variations/InputExampleActionDropdown.js

```diff
@@ -7,13 +7,13 @@ const options = [
   { text: 'Entire Site', value: 'site' },
 ]
 
-const InputActionDropdown = () => (
+const InputExampleActionDropdown = () => (
   <Input
-    action={<Dropdown basic floating options={options} defaultValue='page' />}
+    action={<Dropdown selection basic floating options={options} defaultValue='page' />}
```

Comment:
This should not be a`selection` Dropdown, reference: http://semantic-ui.com/elements/input#action